config/routes.rb

```diff
@@ -1,27 +1,23 @@
 ETM::Application.routes.draw do
-
-  # Temporary: Redirect RLI scenarios to beta scenarios
-  match '/rli/ser-scenario-2023'                    => redirect { 'http://pro.et-model.com/scenarios/193349' }
-  match '/rli/80-procent-co2-reductiescenario-2050' => redirect { 'http://pro.et-model.com/scenarios/423879' }
-  match '/rli/95-procent-co2-reductiescenario-2050' => redirect { 'http://pro.et-model.com/scenarios/423882' }
-
-  # match '/help' => 'articles#index', as: :help
+  REDIRECTS.each_pair do |url, scenario_id|
+    match url => redirect { |_, _|
```

Comment:
Maybe a bit of a style question, but when *all* the block parameters are unused, wouldn't it be better to leave them out all together?

```Ruby
# before
REDIRECTS.each_pair do |url, scenario_id|
  match url => redirect { |_, _|
    "http://pro.et-model.com/scenarios/#{ scenario_id }"
  }
end

# after
REDIRECTS.each_pair do |url, scenario_id|
  match url => redirect {
    "http://pro.et-model.com/scenarios/#{ scenario_id }"
  }
end
```